must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -11,7 +11,10 @@ fi
 SINCE_TIME=$2
 
 # Make a globle variable for namespace
-INSTALL_NAMESPACE=openshift-storage
+INSTALL_NAMESPACES=$(oc get storagecluster -A --no-headers | awk '{print $1}')
+PRODUCT_NAMESPACE=$(oc get managedFusionOffering -A --no-headers | awk '{print $1}')
```

Comment:
@yati1998 There can be multiple ManagedFusionOffering CR in different namespaces in a cluster.